<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Hanqing  Liu</title>
    <meta name="author" content="Hanqing  Liu">
    <meta name="description" content="I am a scientist studying gene regulation in the brain.
">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    <!-- Styles -->
    
    <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://lhqing.github.io/">

    <!-- Dark Mode -->
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav sticky-bottom-footer">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <!-- Social Icons -->
          <div class="navbar-brand social">
            <a href="mailto:%68%61%6E%71%69%6E%67%6C%69%75@%66%61%73.%68%61%72%76%61%72%64.%65%64%75" title="email"><i class="fas fa-envelope"></i></a>
            <a href="https://orcid.org/0000-0002-5114-6048" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a>
            <a href="https://scholar.google.com/citations?user=dh8c-Q4AAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a>
            <a href="https://github.com/lhqing" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a>
            <a href="https://www.linkedin.com/in/hanqing-liu-a63162155" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fab fa-linkedin"></i></a>
            <a href="https://twitter.com/Hanq_Liu" title="Twitter" rel="external nofollow noopener" target="_blank"><i class="fab fa-twitter"></i></a>
            

          </div>
          
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item active">
                <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/publications/">publications</a>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      <!-- about.html -->
<div class="post">
    <header class="post-header">
        <h1 class="post-title">
            <span class="font-weight-bold">Hanqing</span>  Liu
        </h1>
        <p class="desc">Junior Fellow, Society of Fellows, Harvard University <br> Postdoctoral Fellow, Greenberg Lab, Harvard Medical School</p>
    </header>

    <article>
        <div class="profile float-right">

            <figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/prof_pic-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/prof_pic-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/prof_pic-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/prof_pic.jpg" class="img-fluid z-depth-1 rounded" width="auto" height="auto" alt="prof_pic.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>

        </div>

        <div class="clearfix">
            <p>Hi, I am Hanqing Liu (刘翰青).</p>

<p>I am currently a Harvard <a href="https://socfell.fas.harvard.edu/about" rel="external nofollow noopener" target="_blank">Junior Fellow</a> 
working with the <a href="https://greenberg.hms.harvard.edu/" rel="external nofollow noopener" target="_blank">Greenberg lab</a> 
at <a href="https://hms.harvard.edu/" rel="external nofollow noopener" target="_blank">Harvard Medical School</a>.</p>

<p>Before joining Harvard, I was a postdoctoral fellow in the <a href="https://ecker.salk.edu/" rel="external nofollow noopener" target="_blank">Ecker lab</a> at the
<a href="https://www.salk.edu/" rel="external nofollow noopener" target="_blank">Salk Institute for Biological Studies</a>,
where I got my PhD in Biology from UCSD under the guidance of
<a href="https://www.hhmi.org/scientists/joseph-r-ecker" rel="external nofollow noopener" target="_blank">Dr. Joseph Ecker</a>.
My doctoral research focuses on developing single-cell multi-omic technologies and analytical frameworks
to investigate and decipher the epigenomic diversity of mammalian brains across spatial and temporal dimensions.</p>

<p>I got my BSc in Biotechnology from the <a href="https://en.wikipedia.org/wiki/Zhejiang_University" rel="external nofollow noopener" target="_blank">Zhejiang University</a> in China,
graduating with honors from the Chu Kochen Honors College. 
I also had one year clinical training at the Zhejiang University School of Medicine, 
before I decided to change my career path to basic research.</p>

<p>I grow up in <a href="https://en.wikipedia.org/wiki/Hangzhou" rel="external nofollow noopener" target="_blank">Hangzhou</a>, the capital city of Zhejiang Province in China. 
I enjoy taking photos and going to classical music concerts in my spare time.</p>

        </div>

        <!-- News -->
        

        <!-- Latest posts -->
        

        <div class="cv">
            
            <div>
                <h3 class="font-weight-medium">Education</h3>
                <div>
                    
                    <ul class="card-text font-weight-light list-group list-group-flush">
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2022
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Doctor of Philosophy, Biology</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">University of California, San Diego, California, United States</h6>
            
            
              <ul class="items">
                
                  <li>
                    
                      <span class="item">PhD advisor, Prof. Joseph R. Ecker</span>
                    
                  </li>
                
                  <li>
                    
                      <span class="item">Dissertation Title, "DNA methylation landscape of the mouse and human brain at single-cell resolution"</span>
                    
                  </li>
                
              </ul>
            
            
          </div>
        </div>
      </li>
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2016
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Bachelor of Science, Biotechnology</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Zhejiang University, Zhejiang, China</h6>
            
            
            
          </div>
        </div>
      </li>
    
    </ul>

                    
                </div>
            </div>
            
            <div>
                <h3 class="font-weight-medium">Experience</h3>
                <div>
                    
                    <ul class="card-text font-weight-light list-group list-group-flush">
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2023 -
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Junior Fellow</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Society of Fellows, Harvard University</h6>
            
            
            
          </div>
        </div>
      </li>
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2023 -
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Postdoc Fellow, Michael Greenberg Lab</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Harvard Medical School</h6>
            
            
            
          </div>
        </div>
      </li>
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2022 - 2023
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Postdoc Fellow, Joseph Ecker Lab</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Salk Institute for Biological Studies</h6>
            
            
            
          </div>
        </div>
      </li>
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2017 - 2022
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Graduate Student, Joseph Ecker Lab</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Salk Institute for Biological Studies &amp; University of California, San Diego</h6>
            
            
            
          </div>
        </div>
      </li>
    
    </ul>

                    
                </div>
            </div>
            
            <div>
                <h3 class="font-weight-medium">Honors and Awards</h3>
                <div>
                    
                    <ul class="card-text font-weight-light list-group list-group-flush">
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2023
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Junior Fellowship</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Society of Fellows, Harvard University</h6>
            
            
            
          </div>
        </div>
      </li>
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2022
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Pioneer Fund Postdoctoral Scholar Award</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Salk Institute For Biological Studies</h6>
            
            
            
          </div>
        </div>
      </li>
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2021
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">Founding Faculty Award</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Division of Biological Science, UCSD</h6>
            
            
            
          </div>
        </div>
      </li>
    
      <li class="list-group-item">
        <div class="row">
          
            <div class="col-xs-2 cl-sm-2 col-md-2 text-center" style="width: 75px;">
              <span class="badge font-weight-bold primary-color-dark text-uppercase align-middle" style="min-width: 75px;">
                2020
              </span>
            </div>
          
          <div class="col-xs-10 cl-sm-10 col-md-10 mt-2 mt-md-0">
            
            <h6 class="title font-weight-bold ml-1 ml-md-4">David V. Goeddel Endowed Graduate Fellowship</h6>
            
            
            <h6 class="ml-1 ml-md-4" style="font-size: 0.95rem;">Division of Biological Science, UCSD</h6>
            
            
            
          </div>
        </div>
      </li>
    
    </ul>

                    
                </div>
            </div>
            
            <div>
                <h3 class="font-weight-medium mt-4"><a href="/publications/" style="color: inherit;">Selected Publications</a></h3>
                <div>
          <div class="publications">
            <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/bioRxiv-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/bioRxiv-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/bioRxiv-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/bioRxiv.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="bioRxiv.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Liu2023-ri" class="col-sm-8">
        <!-- Title -->
        <div class="title">Single-cell DNA Methylome and 3D Multi-omic Atlas of the
              Adult Mouse Brain</div>
        <!-- Author -->
        <div class="author">
            

            <em><strong>Hanqing Liu</strong></em>, Qiurui Zeng, Jingtian Zhou, Anna Bartlett, Bang-An Wang, Peter Berube, Wei Tian, Mia Kenworthy, Jordan Altshul, Joseph R Nery, Huaming Chen, Rosa G Castanon, Songpeng Zu, Yang Eric Li, Jacinta Lucero, Julia K Osteen, Antonio Pinto-Duarte, Jasper Lee, Jon Rink, Silvia Cho, Nora Emerson, Michael Nunn, Carolyn O’Connor, Zizhen Yao, Bosiljka Tasic, Kimberly A Smith, Hongkui Zeng, Chongyuan Luo, Jesse R Dixon, Bing Ren, M Margarita Behrens, and Joseph R Ecker</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Apr 2023
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.biorxiv.org/content/10.1101/2023.04.16.536509v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2023.04.16.536509"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1101/2023.04.16.536509" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Cytosine DNA methylation is essential in brain development and
              has been implicated in various neurological disorders. A
              comprehensive understanding of DNA methylation diversity across
              the entire brain in the context of the brain’s 3D spatial
              organization is essential for building a complete molecular atlas
              of brain cell types and understanding their gene regulatory
              landscapes. To this end, we employed optimized single-nucleus
              methylome (snmC-seq3) and multi-omic (snm3C-seq) sequencing
              technologies to generate 301,626 methylomes and 176,003 chromatin
              conformation/methylome joint profiles from 117 dissected regions
              throughout the adult mouse brain. Using iterative clustering and
              integrating with companion whole-brain transcriptome and
              chromatin accessibility datasets, we constructed a
              methylation-based cell type taxonomy that contains 4,673 cell
              groups and 261 cross-modality-annotated subclasses. We identified
              millions of differentially methylated regions (DMRs) across the
              genome, representing potential gene regulation elements. Notably,
              we observed spatial cytosine methylation patterns on both genes
              and regulatory elements in cell types within and across brain
              regions. Brain-wide multiplexed error-robust fluorescence in situ
              hybridization (MERFISH) data validated the association of this
              spatial epigenetic diversity with transcription and allowed the
              mapping of the DNA methylation and topology information into
              anatomical structures more precisely than our dissections.
              Furthermore, multi-scale chromatin conformation diversities occur
              in important neuronal genes, highly associated with DNA
              methylation and transcription changes. Brain-wide cell type
              comparison allowed us to build a regulatory model for each gene,
              linking transcription factors, DMRs, chromatin contacts, and
              downstream genes to establish regulatory networks. Finally,
              intragenic DNA methylation and chromatin conformation patterns
              predicted alternative gene isoform expression observed in a
              companion whole-brain SMART-seq dataset. Our study establishes
              the first brain-wide, single-cell resolution DNA methylome and 3D
              multi-omic atlas, providing an unparalleled resource for
              comprehending the mouse brain’s cellular-spatial and regulatory
              genome diversity.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Cell Genomics Volumn 2 Issue 3.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Luo2022-dh" class="col-sm-8">
        <!-- Title -->
        <div class="title">Single nucleus multi-omics identifies human cortical cell
               regulatory genome diversity</div>
        <!-- Author -->
        <div class="author">
            

            Chongyuan Luo*, <em><strong>Hanqing Liu*</strong></em>, Fangming Xie*, Ethan J Armand, Kimberly Siletti, Trygve E Bakken, Rongxin Fang, Wayne I Doyle, Tim Stuart, Rebecca D Hodge, Lijuan Hu, Bang-An Wang, Zhuzhu Zhang, Sebastian Preissl, Dong-Sung Lee, Jingtian Zhou, Sheng-Yong Niu, Rosa Castanon, Anna Bartlett, Angeline Rivkin, Xinxin Wang, Jacinta Lucero, Joseph R Nery, David A Davis, Deborah C Mash, Rahul Satija, Jesse R Dixon, Sten Linnarsson, Ed Lein, M Margarita Behrens, Bing Ren, Eran A Mukamel, and Joseph R Ecker</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Cell Genomics</em>, Mar 2022
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.sciencedirect.com/science/article/pii/S2666979X22000271" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.xgen.2022.100107"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1016/j.xgen.2022.100107" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>SummarySingle-cell technologies measure unique cellular
               signatures but are typically limited to a single modality.
               Computational approaches allow the fusion of diverse single-cell
               data types, but their efficacy is difficult to validate in the
               absence of authentic multi-omic measurements. To comprehensively
               assess the molecular phenotypes of single cells, we devised
               single-nucleus methylcytosine, chromatin accessibility, and
               transcriptome sequencing (snmCAT-seq) and applied it to
               postmortem human frontal cortex tissue. We developed a
               cross-validation approach using multi-modal information to
               validate fine-grained cell types and assessed the effectiveness
               of computational data fusion methods. Correlation analysis in
               individual cells revealed distinct relations between methylation
               and gene expression. Our integrative approach enabled joint
               analyses of the methylome, transcriptome, chromatin
               accessibility, and conformation for 63 human cortical cell
               types. We reconstructed regulatory lineages for cortical cell
               populations and found specific enrichment of genetic risk for
               neuropsychiatric traits, enabling the prediction of cell types
               that are associated with diseases.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Liu2021-xp" class="col-sm-8">
        <!-- Title -->
        <div class="title">DNA methylation atlas of the mouse brain at single-cell
              resolution</div>
        <!-- Author -->
        <div class="author">
            

            <em><strong>Hanqing Liu*</strong></em>, Jingtian Zhou*, Wei Tian, Chongyuan Luo, Anna Bartlett, Andrew Aldridge, Jacinta Lucero, Julia K Osteen, Joseph R Nery, Huaming Chen, Angeline Rivkin, Rosa G Castanon, Ben Clock, Yang Eric Li, Xiaomeng Hou, Olivier B Poirion, Sebastian Preissl, Antonio Pinto-Duarte, Carolyn O’Connor, Lara Boggeman, Conor Fitzpatrick, Michael Nunn, Eran A Mukamel, Zhuzhu Zhang, Edward M Callaway, Bing Ren, Jesse R Dixon, M Margarita Behrens, and Joseph R Ecker</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-020-03182-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-020-03182-8"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-020-03182-8" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Mammalian brain cells show remarkable diversity in gene
              expression, anatomy and function, yet the regulatory DNA
              landscape underlying this extensive heterogeneity is poorly
              understood. Here we carry out a comprehensive assessment of the
              epigenomes of mouse brain cell types by applying single-nucleus
              DNA methylation sequencing1,2 to profile 103,982 nuclei
              (including 95,815 neurons and 8,167 non-neuronal cells) from 45
              regions of the mouse cortex, hippocampus, striatum, pallidum and
              olfactory areas. We identified 161 cell clusters with distinct
              spatial locations and projection targets. We constructed
              taxonomies of these epigenetic types, annotated with signature
              genes, regulatory elements and transcription factors. These
              features indicate the potential regulatory landscape supporting
              the assignment of putative cell types and reveal repetitive usage
              of regulators in excitatory and inhibitory cells for determining
              subtypes. The DNA methylation landscape of excitatory neurons in
              the cortex and hippocampus varied continuously along spatial
              gradients. Using this deep dataset, we constructed an artificial
              neural network model that precisely predicts single neuron
              cell-type identity and brain area spatial location. Integration
              of high-resolution DNA methylomes with single-nucleus chromatin
              accessibility data3 enabled prediction of high-confidence
              enhancer-gene interactions for all identified cell types, which
              were subsequently validated by cell-type-specific chromatin
              conformation capture experiments4. By combining multi-omic
              datasets (DNA methylation, chromatin contacts, and open
              chromatin) from single nuclei and annotating the regulatory
              genome of hundreds of cell types in the mouse brain, our DNA
              methylation atlas establishes the epigenetic basis for neuronal
              diversity and spatial organization throughout the mouse cerebrum.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Yao2021-si" class="col-sm-8">
        <!-- Title -->
        <div class="title">A transcriptomic and epigenomic cell atlas of the mouse primary
              motor cortex</div>
        <!-- Author -->
        <div class="author">
            

            Zizhen Yao*, <em><strong>Hanqing Liu*</strong></em>, Fangming Xie*, Stephan Fischer*, Ricky S Adkins, Andrew I Aldridge, Seth A Ament, Anna Bartlett, M Margarita Behrens, Koen Berge, Darren Bertagnolli, Hector Roux Bézieux, Tommaso Biancalani, A Sina Booeshaghi, Héctor Corrada Bravo, Tamara Casper, Carlo Colantuoni, Jonathan Crabtree, Heather Creasy, Kirsten Crichton, Megan Crow, Nick Dee, Elizabeth L Dougherty, Wayne I Doyle, Sandrine Dudoit, Rongxin Fang, Victor Felix, Olivia Fong, Michelle Giglio, Jeff Goldy, Mike Hawrylycz, Brian R Herb, Ronna Hertzano, Xiaomeng Hou, Qiwen Hu, Jayaram Kancherla, Matthew Kroll, Kanan Lathia, Yang Eric Li, Jacinta D Lucero, and
            <span class="more-authors" title="click to view 43 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '43 more authors' ? 'Chongyuan Luo, Anup Mahurkar, Delissa McMillen, Naeem M Nadaf, Joseph R Nery, Thuc Nghi Nguyen, Sheng-Yong Niu, Vasilis Ntranos, Joshua Orvis, Julia K Osteen, Thanh Pham, Antonio Pinto-Duarte, Olivier Poirion, Sebastian Preissl, Elizabeth Purdom, Christine Rimorin, Davide Risso, Angeline C Rivkin, Kimberly Smith, Kelly Street, Josef Sulc, Valentine Svensson, Michael Tieu, Amy Torkelson, Herman Tung, Eeshit Dhaval Vaishnav, Charles R Vanderburg, Cindy Velthoven, Xinxin Wang, Owen R White, Z Josh Huang, Peter V Kharchenko, Lior Pachter, John Ngai, Aviv Regev, Bosiljka Tasic, Joshua D Welch, Jesse Gillis, Evan Z Macosko, Bing Ren, Joseph R Ecker, Hongkui Zeng, Eran A Mukamel' : '43 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '1');
              ">43 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-021-03500-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-021-03500-8"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03500-8" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Single-cell transcriptomics can provide quantitative molecular
              signatures for large, unbiased samples of the diverse cell types
              in the brain1-3. With the proliferation of multi-omics datasets,
              a major challenge is to validate and integrate results into a
              biological understanding of cell-type organization. Here we
              generated transcriptomes and epigenomes from more than 500,000
              individual cells in the mouse primary motor cortex, a structure
              that has an evolutionarily conserved role in locomotion. We
              developed computational and statistical methods to integrate
              multimodal data and quantitatively validate cell-type
              reproducibility. The resulting reference atlas-containing over 56
              neuronal cell types that are highly replicable across analysis
              methods, sequencing technologies and modalities-is a
              comprehensive molecular and genomic account of the diverse
              neuronal and non-neuronal cell types in the mouse primary motor
              cortex. The atlas includes a population of excitatory neurons
              that resemble pyramidal cells in layer 4 in other cortical
              regions4. We further discovered thousands of concordant marker
              genes and gene regulatory elements for these cell types. Our
              results highlight the complex molecular regulation of cell types
              in the brain and will directly enable the design of reagents to
              target specific cell types in the mouse primary motor cortex for
              functional analysis.</p>
        </div>
    </div>
</div>
</li>
</ol>
          </div>

                </div>
            </div>
        </div>


        <!-- Social -->
        <div class="social">
            <div class="contact-icons">
                <a href="mailto:%68%61%6E%71%69%6E%67%6C%69%75@%66%61%73.%68%61%72%76%61%72%64.%65%64%75" title="email"><i class="fas fa-envelope"></i></a>
            <a href="https://orcid.org/0000-0002-5114-6048" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a>
            <a href="https://scholar.google.com/citations?user=dh8c-Q4AAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a>
            <a href="https://github.com/lhqing" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a>
            <a href="https://www.linkedin.com/in/hanqing-liu-a63162155" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fab fa-linkedin"></i></a>
            <a href="https://twitter.com/Hanq_Liu" title="Twitter" rel="external nofollow noopener" target="_blank"><i class="fab fa-twitter"></i></a>
            

            </div>

            <div class="contact-note">
                
            </div>

        </div>
    </article>

</div>

    </div>

    <!-- Footer -->    <footer class="sticky-bottom mt-5">
      <div class="container">
        © Copyright 2023 Hanqing  Liu. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>.

      </div>
    </footer>


    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-RC1D7R1NBW"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){ window.dataLayer.push(arguments); }
    gtag('js', new Date());
    gtag('config', 'G-RC1D7R1NBW');
  </script>
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar,
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
