<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>publications | Hanqing  Liu</title>
    <meta name="author" content="Hanqing  Liu">
    <meta name="description" content="My full publications in reversed chronological order.
">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    <!-- Styles -->
    
    <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://lhqing.github.io/publications/">
    
    <!-- Dark Mode -->
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav sticky-bottom-footer">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Hanqing </span>Liu</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">publications</h1>
            <p class="post-description">My full publications in reversed chronological order.
</p>
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">
<h2 class="year">2023</h2>
<ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/bioRxiv-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/bioRxiv-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/bioRxiv-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/bioRxiv.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="bioRxiv.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Liu2023-ri" class="col-sm-8">
        <!-- Title -->
        <div class="title">Single-cell DNA Methylome and 3D Multi-omic Atlas of the
              Adult Mouse Brain</div>
        <!-- Author -->
        <div class="author">
            

            <em><strong>Hanqing Liu</strong></em>, Qiurui Zeng, Jingtian Zhou, Anna Bartlett, Bang-An Wang, Peter Berube, Wei Tian, Mia Kenworthy, Jordan Altshul, Joseph R Nery, Huaming Chen, Rosa G Castanon, Songpeng Zu, Yang Eric Li, Jacinta Lucero, Julia K Osteen, Antonio Pinto-Duarte, Jasper Lee, Jon Rink, Silvia Cho, Nora Emerson, Michael Nunn, Carolyn O’Connor, Zizhen Yao, Bosiljka Tasic, Kimberly A Smith, Hongkui Zeng, Chongyuan Luo, Jesse R Dixon, Bing Ren, M Margarita Behrens, and Joseph R Ecker</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Apr 2023
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.biorxiv.org/content/10.1101/2023.04.16.536509v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2023.04.16.536509v1"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1101/2023.04.16.536509v1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Cytosine DNA methylation is essential in brain development and
              has been implicated in various neurological disorders. A
              comprehensive understanding of DNA methylation diversity across
              the entire brain in the context of the brain’s 3D spatial
              organization is essential for building a complete molecular atlas
              of brain cell types and understanding their gene regulatory
              landscapes. To this end, we employed optimized single-nucleus
              methylome (snmC-seq3) and multi-omic (snm3C-seq) sequencing
              technologies to generate 301,626 methylomes and 176,003 chromatin
              conformation/methylome joint profiles from 117 dissected regions
              throughout the adult mouse brain. Using iterative clustering and
              integrating with companion whole-brain transcriptome and
              chromatin accessibility datasets, we constructed a
              methylation-based cell type taxonomy that contains 4,673 cell
              groups and 261 cross-modality-annotated subclasses. We identified
              millions of differentially methylated regions (DMRs) across the
              genome, representing potential gene regulation elements. Notably,
              we observed spatial cytosine methylation patterns on both genes
              and regulatory elements in cell types within and across brain
              regions. Brain-wide multiplexed error-robust fluorescence in situ
              hybridization (MERFISH) data validated the association of this
              spatial epigenetic diversity with transcription and allowed the
              mapping of the DNA methylation and topology information into
              anatomical structures more precisely than our dissections.
              Furthermore, multi-scale chromatin conformation diversities occur
              in important neuronal genes, highly associated with DNA
              methylation and transcription changes. Brain-wide cell type
              comparison allowed us to build a regulatory model for each gene,
              linking transcription factors, DMRs, chromatin contacts, and
              downstream genes to establish regulatory networks. Finally,
              intragenic DNA methylation and chromatin conformation patterns
              predicted alternative gene isoform expression observed in a
              companion whole-brain SMART-seq dataset. Our study establishes
              the first brain-wide, single-cell resolution DNA methylome and 3D
              multi-omic atlas, providing an unparalleled resource for
              comprehending the mouse brain’s cellular-spatial and regulatory
              genome diversity.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/bioRxiv-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/bioRxiv-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/bioRxiv-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/bioRxiv.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="bioRxiv.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Zemke2023-rx" class="col-sm-8">
        <!-- Title -->
        <div class="title">Comparative single cell epigenomic analysis of gene regulatory
              programs in the rodent and primate neocortex</div>
        <!-- Author -->
        <div class="author">
            

            Nathan R Zemke, Ethan J Armand, Wenliang Wang, Seoyeon Lee, Jingtian Zhou, Yang Eric Li, <em><strong>Hanqing Liu</strong></em>, Wei Tian, Joseph R Nery, Rosa G Castanon, Anna Bartlett, Julia K Osteen, Daofeng Li, Xiaoyu Zhuo, Vincent Xu, Michael Miller, Fenna M Krienen, Qiangge Zhang, Naz Taskin, Jonathan Ting, Guoping Feng, Steven A McCarroll, Edward M Callaway, Ting Wang, M Margarita Behrens, Ed S Lein, Joseph R Ecker, and Bing Ren</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Apr 2023
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.biorxiv.org/content/10.1101/2023.04.08.536119v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2023.04.08.536119"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1101/2023.04.08.536119" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Sequence divergence of cis- regulatory elements drives
              species-specific traits, but how this manifests in the evolution
              of the neocortex at the molecular and cellular level remains to
              be elucidated. We investigated the gene regulatory programs in
              the primary motor cortex of human, macaque, marmoset, and mouse
              with single-cell multiomics assays, generating gene expression,
              chromatin accessibility, DNA methylome, and chromosomal
              conformation profiles from a total of over 180,000 cells. For
              each modality, we determined species-specific, divergent, and
              conserved gene expression and epigenetic features at multiple
              levels. We find that cell type-specific gene expression evolves
              more rapidly than broadly expressed genes and that epigenetic
              status at distal candidate cis -regulatory elements (cCREs)
              evolves faster than promoters. Strikingly, transposable elements
              (TEs) contribute to nearly 80% of the human-specific cCREs in
              cortical cells. Through machine learning, we develop
              sequence-based predictors of cCREs in different species and
              demonstrate that the genomic regulatory syntax is highly
              preserved from rodents to primates. Lastly, we show that
              epigenetic conservation combined with sequence similarity helps
              uncover functional cis -regulatory elements and enhances our
              ability to interpret genetic variants contributing to
              neurological disease and traits.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 abbr"></div>

    <!-- Entry bib key -->
    <div id="Zhou2023-di" class="col-sm-8">
        <!-- Title -->
        <div class="title">Brain-wide Correspondence Between Neuronal Epigenomics and
             Long-Distance Projections</div>
        <!-- Author -->
        <div class="author">
            

            Jingtian Zhou, Zhuzhu Zhang, May Wu, <em><strong>Hanqing Liu</strong></em>, Yan Pang, Anna Bartlett, Angeline Rivkin, Will N Lagos, Elora Williams, Cheng-Ta Lee, Paula Assakura Miyazaki, Andrew Aldridge, Qiurui Zeng, J L Angelo Salinda, Naomi Claffey, Michelle Liem, Conor Fitzpatrick, Lara Boggeman, Zizhen Yao, Kimberly A Smith, Bosiljka Tasic, Jordan Altshul, Mia A Kenworthy, Cynthia Valadon, Joseph R Nery, Rosa G Castanon, Neelakshi S Patne, Minh Vu, Mohammad Rashid, Matthew Jacobs, Tony Ito, Julia Osteen, Nora Emerson, Jasper Lee, Silvia Cho, Jon Rink, Sharon Huang, António Pinto-Duartec, Bertha Dominguez, Jared B Smith, and
            <span class="more-authors" title="click to view 8 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '8 more authors' ? 'Carolyn O’Connor, Hongkui Zeng, Kuo-Fen Lee, Eran A Mukamel, Xin Jin, M Margarita Behrens, Joseph R Ecker, Edward M Callaway' : '8 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '1');
              ">8 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Apr 2023
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span>
            <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Single-cell genetic and epigenetic analyses parse the brain’s
    billions of neurons into thousands of “cell-type” clusters, each residing
    in different brain structures. Many of these cell types mediate their unique
    functions by virtue of targeted long-distance axonal projections to allow
    interactions between specific cell types. Here we have used Epi-Retro-Seq to
    link single cell epigenomes and associated cell types to their long-distance
    projections for 33,034 neurons dissected from 32 different source regions
    projecting to 24 different targets (225 source→target combinations) across the
    whole mouse brain. We highlight uses of this large data set for interrogating
    both overarching principles relating projection cell types to their transcriptomic
    and epigenomic properties and for addressing and developing specific hypotheses
    about cell types and connections as they relate to genetics. We provide an
    overall synthesis of the data set with 926 statistical comparisons of the
    discriminability of neurons projecting to each target for every dissected source
    region. We integrate this dataset into the larger, annotated BICCN cell type
    atlas composed of millions of neurons to link projection cell types to consensus
    clusters. Integration with spatial transcriptomic data further assigns
    projection-enriched clusters to much smaller source regions than afforded by
    the original dissections. We exemplify these capabilities by presenting in-depth
    analyses of neurons with identified projections from the hypothalamus, thalamus,
    hindbrain, amygdala, and midbrain to provide new insights into the properties of
    those cell types, including differentially expressed genes, their associated
    cis-regulatory elements and transcription factor binding motifs, and
    neurotransmitter usage.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 abbr"></div>

    <!-- Entry bib key -->
    <div id="Zu2023-id" class="col-sm-8">
        <!-- Title -->
        <div class="title">Comprehensive single-cell analysis of chromatin accessibility in
              the adult mouse brain</div>
        <!-- Author -->
        <div class="author">
            

            Songpeng Zu, Yang Eric Li, Kangli Wang, Michael Miller, Yuelai Wang,  Maria, Luisa Amaral, Xiaomeng Hou, Bin Li, Samantha Kuan, <em><strong>Hanqing Liu</strong></em>, Jingtian Zhou, Antonio Pinto-Duarte, Jacinta Lucero, Julia Osteen, Michael Nunn, Kimberly A Smith, Bosiljka Tasic, Zizhen Yao, Hongkui Zeng, Zihan Wang, Jingbo Shang, M Margarita Behrens, Joseph R Ecker, Allen Wang, Sebastian Preissl, and Bing Ren</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Apr 2023
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span>
            <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Abstract: The mouse brain comprises tens of millions of neurons
              and nonneuronal cells organized into numerous circuits and
              structures that are responsible for complex behaviors and
              neurological functions. Advances in single-cell technologies have
              led to the discovery of thousands of brain cell types. However,
              our understanding of the gene regulatory programs in these cell
              types is still limited. Here we report a comprehensive atlas of
              candidate cis-regulatory DNA elements (cCREs) in the adult mouse
              brain, generated through examination of the chromatin
              accessibility in 2.3 million individual brain cells from 117
              anatomical dissections. The atlas includes 1.3 million cCREs and
              their chromatin accessibility across 602 distinct brain cell
              populations, adding over 640,000 new cCREs to the most recent
              such annotation in the mouse genome. The mouse brain cCREs are
              moderately conserved in the human brain. The mouse specific
              cCREs, in particular those identified from a subset of cortical
              excitatory neurons, are strongly enriched for transposable
              elements, suggesting a potential role for transposable elements
              in the emergence of new regulatory programs and neuronal
              diversity. Finally, we infer the gene regulatory networks in over
              200 subclasses of mouse brain cells, and develop deep learning
              models to predict, from DNA sequence alone, the activities of
              gene regulatory elements in different brain cell types. Our
              results provide a resource for analysis of cell-type-specific
              gene regulation programs in both mouse and human brains.</p>
        </div>
    </div>
</div>
</li>
</ol>

<h2 class="year">2022</h2>
<ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/bioRxiv-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/bioRxiv-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/bioRxiv-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/bioRxiv.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="bioRxiv.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Tian2022-zp" class="col-sm-8">
        <!-- Title -->
        <div class="title">Epigenomic complexity of the human brain revealed by single-cell
              DNA methylomes and 3D genome structures</div>
        <!-- Author -->
        <div class="author">
            

            Wei Tian, Jingtian Zhou, Anna Bartlett, Qiurui Zeng, <em><strong>Hanqing Liu</strong></em>, Rosa G Castanon, Mia Kenworthy, Jordan Altshul, Cynthia Valadon, Andrew Aldridge, Joseph R Nery, Huaming Chen, Jiaying Xu, Nicholas D Johnson, Jacinta Lucero, Julia K Osteen, Nora Emerson, Jon Rink, Jasper Lee, Yang Li, Kimberly Siletti, Michelle Liem, Naomi Claffey, Caz O’Connor, Anna Marie Yanny, Julie Nyhus, Nick Dee, Tamara Casper, Nadiya Shapovalova, Daniel Hirschstein, Rebecca Hodge, Boaz P Levi, C Dirk Keene, Sten Linnarsson, Ed Lein, Bing Ren, M Margarita Behrens, and Joseph R Ecker</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Dec 2022
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.biorxiv.org/content/10.1101/2022.11.30.518285" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2022.11.30.518285"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1101/2022.11.30.518285" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Delineating the gene regulatory programs underlying complex cell
              types is fundamental for understanding brain functions in health
              and disease. Here, we comprehensively examine human brain cell
              epigenomes by probing DNA methylation and chromatin conformation
              at single-cell resolution in over 500,000 cells from 46 brain
              regions. We identified 188 cell types and characterized their
              molecular signatures. Integrative analyses revealed concordant
              changes in DNA methylation, chromatin accessibility, chromatin
              organization, and gene expression across cell types, cortical
              areas, and basal ganglia structures. With these resources, we
              developed scMCodes that reliably predict brain cell types using
              their methylation status at select genomic sites. This multimodal
              epigenomic brain cell atlas provides new insights into the
              complexity of cell type-specific gene regulation in the adult
              human brain. ### Competing Interest Statement J.R.E is a
              member of the scientific advisor for Zymo Research and Ionis.
              B.R. is a co-founder and consultant of Arima Genomics Inc. and
              co-founder of Epigenome Technologies.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/bioRxiv-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/bioRxiv-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/bioRxiv-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/bioRxiv.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="bioRxiv.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="BICCN_Data_Ecosystem_Collaboration2022-he" class="col-sm-8">
        <!-- Title -->
        <div class="title">The BRAIN Initiative Cell Census Network Data Ecosystem: A
              User’s Guide</div>
        <!-- Author -->
        <div class="author">
            

             BICCN Data Ecosystem Collaboration, Michael J Hawrylycz, Maryann E Martone, Patrick R Hof, Ed S Lein, Aviv Regev, Giorgio A A Ascoli, Jan G Bjaalie, Hong-Wei Dong, Satrajit S Ghosh, Jesse Gillis, Ronna Hertzano, David R Haynor, Yongsoo Kim, Yufeng Liu, Jeremy A Miller, Partha P Mitra, Eran Mukamel, David Osumi-Sutherland, Hanchuan Peng, Patrick L Ray, Raymond Sanchez, Alex Ropelewski, Richard H Scheuermann, Shawn Z K Tan, Timothy Tickle, Hagen Tilgner, Merina Varghese, Brock Wester, Owen White, Brian Aevermann, David Allemang, Seth Ament, Thomas L Athey, Pamela M Baker, Cody Baker, Katherine S Baker, Anita Bandrowski, Prajal Bishwakarma, Ambrose Carr, and
            <span class="more-authors" title="click to view 60 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '60 more authors' ? 'Min Chen, Roni Choudhury, Jonah Cool, Heather Creasy, Florence D’Orazi, Kylee Degatano, Benjamin Dichter, Song-Lin Ding, Tim Dolbeare, Joseph R Ecker, Rongxin Fang, Jean-Christophe Fillion-Robin, Timothy P Fliss, James Gee, Tom Gillespie, Nathan Gouwens, Yaroslav O Halchenko, Nomi Harris, Brian R Herb, Houri Hintiryan, Gregory Hood, Sam Horvath, Dorota Jarecka, Shengdian Jiang, Farzaneh Khajouei, Elizabeth A Kiernan, Huseyin Kir, Lauren Kruse, Changkyu Lee, Boudewijn Lelieveldt, Yang Li, Hanqing Liu, Anup Markuhar, James Mathews, Kaylee L Mathews, Michael I Miller, Tyler Mollenkopf, Shoaib Mufti, Christopher J Mungall, Lydia Ng, Joshua Orvis, Maja A Puchades, Lei Qu, Joseph P Receveur, Bing Ren, Nathan Sjoquist, Brian Staats, Carol L Thompson, Daniel Tward, Cindy T J Velthoven, Quanxin Wang, Fangming Xie, Hua Xu, Zizhen Yao, Zhixi Yun, Hongkui Zeng, Guo-Qiang Zhang, Yun R Zhang, Jim W Zheng, Brian Zingg' : '60 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '1');
              ">60 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Oct 2022
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.biorxiv.org/content/10.1101/2022.10.26.513573v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2022.10.26.513573"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1101/2022.10.26.513573" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Characterizing cellular diversity at different levels of
              biological organization across data modalities is a prerequisite
              to understanding the function of cell types in the brain.
              Classification of neurons is also required to manipulate cell
              types in controlled ways, and to understand their variation and
              vulnerability in brain disorders. The BRAIN Initiative Cell
              Census Network (BICCN) is an integrated network of data
              generating centers, data archives and data standards developers,
              with the goal of systematic multimodal brain cell type profiling
              and characterization. Emphasis of the BICCN is on the whole mouse
              brain and demonstration of prototypes for human and non-human
              primate (NHP) brains. Here, we provide a guide to the cellular
              and spatial approaches employed, and to accessing and using the
              BICCN data and its extensive resources, including the BRAIN Cell
              Data Center (BCDC) which serves to manage and integrate data
              across the ecosystem. We illustrate the power of the BICCN data
              ecosystem through vignettes highlighting several BICCN analysis
              and visualization tools. Finally, we present emerging standards
              that have been developed or adopted by the BICCN toward FAIR
              ([Wilkinson et al. 2016a][1]) neuroscience. The combined BICCN
              ecosystem provides a comprehensive resource for the exploration
              and analysis of cell types in the brain. ### Competing
              Interest Statement Aviv Regev is a co-founder and equity holder
              of Celsius Therapeutics, an equity holder in Immunitas
              Therapeutics and, until 31 July 2020, was a scientific advisory
              board member of Thermo Fisher Scientific, Syros Pharmaceuticals,
              Asimov, and Neogene Therapeutics. From 1 August 2020, AR is an
              employee of Genentech and has equity in Roche. AR is a named
              inventor on multiple patents related to single cell and spatial
              genomics filed by or issued to the Broad Institute. [1]: #ref-72</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Cell%20Genomics%20Volumn%202%20Issue%203.gif" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Cell Genomics Volumn 2 Issue 3.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Luo2022-dh" class="col-sm-8">
        <!-- Title -->
        <div class="title">Single nucleus multi-omics identifies human cortical cell
               regulatory genome diversity</div>
        <!-- Author -->
        <div class="author">
            

            Chongyuan Luo*, <em><strong>Hanqing Liu*</strong></em>, Fangming Xie*, Ethan J Armand, Kimberly Siletti, Trygve E Bakken, Rongxin Fang, Wayne I Doyle, Tim Stuart, Rebecca D Hodge, Lijuan Hu, Bang-An Wang, Zhuzhu Zhang, Sebastian Preissl, Dong-Sung Lee, Jingtian Zhou, Sheng-Yong Niu, Rosa Castanon, Anna Bartlett, Angeline Rivkin, Xinxin Wang, Jacinta Lucero, Joseph R Nery, David A Davis, Deborah C Mash, Rahul Satija, Jesse R Dixon, Sten Linnarsson, Ed Lein, M Margarita Behrens, Bing Ren, Eran A Mukamel, and Joseph R Ecker</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Cell Genomics</em>, Mar 2022
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.sciencedirect.com/science/article/pii/S2666979X22000271" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.xgen.2022.100107"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1016/j.xgen.2022.100107" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>SummarySingle-cell technologies measure unique cellular
               signatures but are typically limited to a single modality.
               Computational approaches allow the fusion of diverse single-cell
               data types, but their efficacy is difficult to validate in the
               absence of authentic multi-omic measurements. To comprehensively
               assess the molecular phenotypes of single cells, we devised
               single-nucleus methylcytosine, chromatin accessibility, and
               transcriptome sequencing (snmCAT-seq) and applied it to
               postmortem human frontal cortex tissue. We developed a
               cross-validation approach using multi-modal information to
               validate fine-grained cell types and assessed the effectiveness
               of computational data fusion methods. Correlation analysis in
               individual cells revealed distinct relations between methylation
               and gene expression. Our integrative approach enabled joint
               analyses of the methylome, transcriptome, chromatin
               accessibility, and conformation for 63 human cortical cell
               types. We reconstructed regulatory lineages for cortical cell
               populations and found specific enrichment of genetic risk for
               neuropsychiatric traits, enabling the prediction of cell types
               that are associated with diseases.</p>
        </div>
    </div>
</div>
</li>
</ol>

<h2 class="year">2021</h2>
<ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/bioRxiv-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/bioRxiv-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/bioRxiv-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/bioRxiv.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="bioRxiv.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Xie2021-pn" class="col-sm-8">
        <!-- Title -->
        <div class="title">Robust enhancer-gene regulation identified by single-cell
              transcriptomes and epigenomes</div>
        <!-- Author -->
        <div class="author">
            

            Fangming Xie, Ethan J Armand, Zizhen Yao, <em><strong>Hanqing Liu</strong></em>, Anna Bartlett, M Margarita Behrens, Yang Eric Li, Jacinta D Lucero, Chongyuan Luo, Joseph R Nery, Antonio Pinto-Duarte, Olivier Poirion, Sebastian Preissl, Angeline C Rivkin, Bosiljka Tasic, Hongkui Zeng, Bing Ren, Joseph R Ecker, and Eran A Mukamel</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.biorxiv.org/content/10.1101/2021.10.25.465795v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1101/2021.10.25.465795"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1101/2021.10.25.465795" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Integrating single-cell transcriptomes and epigenomes across
              diverse cell types can link genes with the cis -regulatory
              elements (CREs) that control expression. Gene co-expression
              across cell types confounds simple correlation-based analysis and
              results in high false prediction rates. We developed a procedure
              that controls for co-expression between genes and integrates
              multiple molecular modalities, and used it to identify &gt;10,000
              gene-CRE pairs that contribute to gene expression programs in
              different cell types in the mouse brain. ### Competing
              Interest Statement The authors have declared no competing
              interest.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/AJHG%20Volume%20108%20Issue%2010.gif-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/AJHG%20Volume%20108%20Issue%2010.gif-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/AJHG%20Volume%20108%20Issue%2010.gif-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/AJHG%20Volume%20108%20Issue%2010.gif" class="preview z-depth-1 rounded" width="auto" height="auto" alt="AJHG Volume 108 Issue 10.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Wu2021-gw" class="col-sm-8">
        <!-- Title -->
        <div class="title">Improved pathogenicity prediction for rare human missense
              variants</div>
        <!-- Author -->
        <div class="author">
            

            Yingzhou Wu, <em><strong>Hanqing Liu</strong></em>, Roujia Li, Song Sun, Jochen Weile, and Frederick P Roth</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Am. J. Hum. Genet.</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.sciencedirect.com/science/article/pii/S0002929721003207" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1016/j.ajhg.2021.08.012"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ajhg.2021.08.012" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>The success of personalized genomic medicine depends on our
              ability to assess the pathogenicity of rare human variants,
              including the important class of missense variation. There are
              many challenges in training accurate computational systems, e.g.,
              in finding the balance between quantity, quality, and bias in the
              variant sets used as training examples and avoiding predictive
              features that can accentuate the effects of bias. Here, we
              describe VARITY, which judiciously exploits a larger reservoir of
              training examples with uncertain accuracy and representativity.
              To limit circularity and bias, VARITY excludes features informed
              by variant annotation and protein identity. To provide a
              rationale for each prediction, we quantified the contribution of
              features and feature combinations to the pathogenicity inference
              of each variant. VARITY outperformed all previous computational
              methods evaluated, identifying at least 10% more pathogenic
              variants at thresholds achieving high (90% precision)
              stringency.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="BRAIN_Initiative_Cell_Census_Network_BICCN2021-qq" class="col-sm-8">
        <!-- Title -->
        <div class="title">A multimodal cell census and atlas of the mammalian primary motor
              cortex</div>
        <!-- Author -->
        <div class="author">
            

             BRAIN Initiative Cell Census Network (BICCN)</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-021-03950-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-021-03950-0"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03950-0" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Here we report the generation of a multimodal cell census and
              atlas of the mammalian primary motor cortex as the initial
              product of the BRAIN Initiative Cell Census Network (BICCN). This
              was achieved by coordinated large-scale analyses of single-cell
              transcriptomes, chromatin accessibility, DNA methylomes,
              spatially resolved single-cell transcriptomes, morphological and
              electrophysiological properties and cellular resolution
              input-output mapping, integrated through cross-modal
              computational analysis. Our results advance the collective
              knowledge and understanding of brain cell-type organization1-5.
              First, our study reveals a unified molecular genetic landscape of
              cortical cell types that integrates their transcriptome, open
              chromatin and DNA methylation maps. Second, cross-species
              analysis achieves a consensus taxonomy of transcriptomic types
              and their hierarchical organization that is conserved from mouse
              to marmoset and human. Third, in situ single-cell transcriptomics
              provides a spatially resolved cell-type atlas of the motor
              cortex. Fourth, cross-modal analysis provides compelling evidence
              for the transcriptomic, epigenomic and gene regulatory basis of
              neuronal phenotypes such as their physiological and anatomical
              properties, demonstrating the biological validity and genomic
              underpinning of neuron types. We further present an extensive
              genetic toolset for targeting glutamatergic neuron types towards
              linking their molecular and developmental identity to their
              circuit function. Together, our results establish a unifying and
              mechanistic framework of neuronal cell-type organization that
              integrates multi-layered molecular genetic and spatial
              information with multi-faceted phenotypic properties.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Zhang2021-jw" class="col-sm-8">
        <!-- Title -->
        <div class="title">Epigenomic diversity of cortical projection neurons in the mouse
              brain</div>
        <!-- Author -->
        <div class="author">
            

            Zhuzhu Zhang, Jingtian Zhou, Pengcheng Tan, Yan Pang, Angeline C Rivkin, Megan A Kirchgessner, Elora Williams, Cheng-Ta Lee, <em><strong>Hanqing Liu</strong></em>, Alexis D Franklin, Paula Assakura Miyazaki, Anna Bartlett, Andrew I Aldridge, Minh Vu, Lara Boggeman, Conor Fitzpatrick, Joseph R Nery, Rosa G Castanon, Mohammad Rashid, Matthew W Jacobs, Tony Ito-Cole, Carolyn O’Connor, António Pinto-Duartec, Bertha Dominguez, Jared B Smith, Sheng-Yong Niu, Kuo-Fen Lee, Xin Jin, Eran A Mukamel, M Margarita Behrens, Joseph R Ecker, and Edward M Callaway</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-021-03223-w" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-021-03223-w"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03223-w" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Neuronal cell types are classically defined by their molecular
              properties, anatomy and functions. Although recent advances in
              single-cell genomics have led to high-resolution molecular
              characterization of cell type diversity in the brain1, neuronal
              cell types are often studied out of the context of their
              anatomical properties. To improve our understanding of the
              relationship between molecular and anatomical features that
              define cortical neurons, here we combined retrograde labelling
              with single-nucleus DNA methylation sequencing to link neural
              epigenomic properties to projections. We examined 11,827 single
              neocortical neurons from 63 cortico-cortical and
              cortico-subcortical long-distance projections. Our results showed
              unique epigenetic signatures of projection neurons that
              correspond to their laminar and regional location and projection
              patterns. On the basis of their epigenomes, intra-telencephalic
              cells that project to different cortical targets could be further
              distinguished, and some layer 5 neurons that project to
              extra-telencephalic targets (L5 ET) formed separate clusters that
              aligned with their axonal projections. Such separation varied
              between cortical areas, which suggests that there are
              area-specific differences in L5 ET subtypes, which were further
              validated by anatomical studies. Notably, a population of
              cortico-cortical projection neurons clustered with L5 ET rather
              than intra-telencephalic neurons, which suggests that a
              population of L5 ET cortical neurons projects to both targets. We
              verified the existence of these neurons by dual retrograde
              labelling and anterograde tracing of cortico-cortical projection
              neurons, which revealed axon terminals in extra-telencephalic
              targets including the thalamus, superior colliculus and pons.
              These findings highlight the power of single-cell epigenomic
              approaches to connect the molecular properties of neurons with
              their anatomical and projection properties.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Liu2021-xp" class="col-sm-8">
        <!-- Title -->
        <div class="title">DNA methylation atlas of the mouse brain at single-cell
              resolution</div>
        <!-- Author -->
        <div class="author">
            

            <em><strong>Hanqing Liu*</strong></em>, Jingtian Zhou*, Wei Tian, Chongyuan Luo, Anna Bartlett, Andrew Aldridge, Jacinta Lucero, Julia K Osteen, Joseph R Nery, Huaming Chen, Angeline Rivkin, Rosa G Castanon, Ben Clock, Yang Eric Li, Xiaomeng Hou, Olivier B Poirion, Sebastian Preissl, Antonio Pinto-Duarte, Carolyn O’Connor, Lara Boggeman, Conor Fitzpatrick, Michael Nunn, Eran A Mukamel, Zhuzhu Zhang, Edward M Callaway, Bing Ren, Jesse R Dixon, M Margarita Behrens, and Joseph R Ecker</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-020-03182-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-020-03182-8"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-020-03182-8" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Mammalian brain cells show remarkable diversity in gene
              expression, anatomy and function, yet the regulatory DNA
              landscape underlying this extensive heterogeneity is poorly
              understood. Here we carry out a comprehensive assessment of the
              epigenomes of mouse brain cell types by applying single-nucleus
              DNA methylation sequencing1,2 to profile 103,982 nuclei
              (including 95,815 neurons and 8,167 non-neuronal cells) from 45
              regions of the mouse cortex, hippocampus, striatum, pallidum and
              olfactory areas. We identified 161 cell clusters with distinct
              spatial locations and projection targets. We constructed
              taxonomies of these epigenetic types, annotated with signature
              genes, regulatory elements and transcription factors. These
              features indicate the potential regulatory landscape supporting
              the assignment of putative cell types and reveal repetitive usage
              of regulators in excitatory and inhibitory cells for determining
              subtypes. The DNA methylation landscape of excitatory neurons in
              the cortex and hippocampus varied continuously along spatial
              gradients. Using this deep dataset, we constructed an artificial
              neural network model that precisely predicts single neuron
              cell-type identity and brain area spatial location. Integration
              of high-resolution DNA methylomes with single-nucleus chromatin
              accessibility data3 enabled prediction of high-confidence
              enhancer-gene interactions for all identified cell types, which
              were subsequently validated by cell-type-specific chromatin
              conformation capture experiments4. By combining multi-omic
              datasets (DNA methylation, chromatin contacts, and open
              chromatin) from single nuclei and annotating the regulatory
              genome of hundreds of cell types in the mouse brain, our DNA
              methylation atlas establishes the epigenetic basis for neuronal
              diversity and spatial organization throughout the mouse cerebrum.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Bakken2021-vl" class="col-sm-8">
        <!-- Title -->
        <div class="title">Comparative cellular analysis of motor cortex in human, marmoset
              and mouse</div>
        <!-- Author -->
        <div class="author">
            

            Trygve E Bakken, Nikolas L Jorstad, Qiwen Hu, Blue B Lake, Wei Tian, Brian E Kalmbach, Megan Crow, Rebecca D Hodge, Fenna M Krienen, Staci A Sorensen, Jeroen Eggermont, Zizhen Yao, Brian D Aevermann, Andrew I Aldridge, Anna Bartlett, Darren Bertagnolli, Tamara Casper, Rosa G Castanon, Kirsten Crichton, Tanya L Daigle, Rachel Dalley, Nick Dee, Nikolai Dembrow, Dinh Diep, Song-Lin Ding, Weixiu Dong, Rongxin Fang, Stephan Fischer, Melissa Goldman, Jeff Goldy, Lucas T Graybuck, Brian R Herb, Xiaomeng Hou, Jayaram Kancherla, Matthew Kroll, Kanan Lathia, Baldur Lew, Yang Eric Li, Christine S Liu, <em><strong>Hanqing Liu</strong></em>, and
            <span class="more-authors" title="click to view 66 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '66 more authors' ? 'Jacinta D Lucero, Anup Mahurkar, Delissa McMillen, Jeremy A Miller, Marmar Moussa, Joseph R Nery, Philip R Nicovich, Sheng-Yong Niu, Joshua Orvis, Julia K Osteen, Scott Owen, Carter R Palmer, Thanh Pham, Nongluk Plongthongkum, Olivier Poirion, Nora M Reed, Christine Rimorin, Angeline Rivkin, William J Romanow, Adriana E Sedeño-Cortés, Kimberly Siletti, Saroja Somasundaram, Josef Sulc, Michael Tieu, Amy Torkelson, Herman Tung, Xinxin Wang, Fangming Xie, Anna Marie Yanny, Renee Zhang, Seth A Ament, M Margarita Behrens, Hector Corrada Bravo, Jerold Chun, Alexander Dobin, Jesse Gillis, Ronna Hertzano, Patrick R Hof, Thomas Höllt, Gregory D Horwitz, C Dirk Keene, Peter V Kharchenko, Andrew L Ko, Boudewijn P Lelieveldt, Chongyuan Luo, Eran A Mukamel, António Pinto-Duarte, Sebastian Preissl, Aviv Regev, Bing Ren, Richard H Scheuermann, Kimberly Smith, William J Spain, Owen R White, Christof Koch, Michael Hawrylycz, Bosiljka Tasic, Evan Z Macosko, Steven A McCarroll, Jonathan T Ting, Hongkui Zeng, Kun Zhang, Guoping Feng, Joseph R Ecker, Sten Linnarsson, Ed S Lein' : '66 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '1');
              ">66 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-021-03465-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-021-03465-8"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03465-8" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>The primary motor cortex (M1) is essential for voluntary
              fine-motor control and is functionally conserved across mammals1.
              Here, using high-throughput transcriptomic and epigenomic
              profiling of more than 450,000 single nuclei in humans, marmoset
              monkeys and mice, we demonstrate a broadly conserved cellular
              makeup of this region, with similarities that mirror evolutionary
              distance and are consistent between the transcriptome and
              epigenome. The core conserved molecular identities of neuronal
              and non-neuronal cell types allow us to generate a cross-species
              consensus classification of cell types, and to infer conserved
              properties of cell types across species. Despite the overall
              conservation, however, many species-dependent specializations are
              apparent, including differences in cell-type proportions, gene
              expression, DNA methylation and chromatin state. Few cell-type
              marker genes are conserved across species, revealing a short list
              of candidate genes and regulatory mechanisms that are responsible
              for conserved features of homologous cell types, such as the
              GABAergic chandelier cells. This consensus transcriptomic
              classification allows us to use patch-seq (a combination of
              whole-cell patch-clamp recordings, RNA sequencing and
              morphological characterization) to identify corticospinal Betz
              cells from layer 5 in non-human primates and humans, and to
              characterize their highly specialized physiology and anatomy.
              These findings highlight the robust molecular underpinnings of
              cell-type diversity in M1 across mammals, and point to the genes
              and regulatory pathways responsible for the functional identity
              of cell types and their species-specific adaptations.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Yao2021-si" class="col-sm-8">
        <!-- Title -->
        <div class="title">A transcriptomic and epigenomic cell atlas of the mouse primary
              motor cortex</div>
        <!-- Author -->
        <div class="author">
            

            Zizhen Yao*, <em><strong>Hanqing Liu*</strong></em>, Fangming Xie*, Stephan Fischer*, Ricky S Adkins, Andrew I Aldridge, Seth A Ament, Anna Bartlett, M Margarita Behrens, Koen Berge, Darren Bertagnolli, Hector Roux Bézieux, Tommaso Biancalani, A Sina Booeshaghi, Héctor Corrada Bravo, Tamara Casper, Carlo Colantuoni, Jonathan Crabtree, Heather Creasy, Kirsten Crichton, Megan Crow, Nick Dee, Elizabeth L Dougherty, Wayne I Doyle, Sandrine Dudoit, Rongxin Fang, Victor Felix, Olivia Fong, Michelle Giglio, Jeff Goldy, Mike Hawrylycz, Brian R Herb, Ronna Hertzano, Xiaomeng Hou, Qiwen Hu, Jayaram Kancherla, Matthew Kroll, Kanan Lathia, Yang Eric Li, Jacinta D Lucero, and
            <span class="more-authors" title="click to view 43 more authors" onclick="
                var element = $(this);
                element.attr('title', '');
                var more_authors_text = element.text() == '43 more authors' ? 'Chongyuan Luo, Anup Mahurkar, Delissa McMillen, Naeem M Nadaf, Joseph R Nery, Thuc Nghi Nguyen, Sheng-Yong Niu, Vasilis Ntranos, Joshua Orvis, Julia K Osteen, Thanh Pham, Antonio Pinto-Duarte, Olivier Poirion, Sebastian Preissl, Elizabeth Purdom, Christine Rimorin, Davide Risso, Angeline C Rivkin, Kimberly Smith, Kelly Street, Josef Sulc, Valentine Svensson, Michael Tieu, Amy Torkelson, Herman Tung, Eeshit Dhaval Vaishnav, Charles R Vanderburg, Cindy Velthoven, Xinxin Wang, Owen R White, Z Josh Huang, Peter V Kharchenko, Lior Pachter, John Ngai, Aviv Regev, Bosiljka Tasic, Joshua D Welch, Jesse Gillis, Evan Z Macosko, Bing Ren, Joseph R Ecker, Hongkui Zeng, Eran A Mukamel' : '43 more authors';
                var cursorPosition = 0;
                var textAdder = setInterval(function(){
                  element.text(more_authors_text.substring(0, cursorPosition + 1));
                  if (++cursorPosition == more_authors_text.length){
                    clearInterval(textAdder);
                  }
              }, '1');
              ">43 more authors</span>
</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-021-03500-8" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-021-03500-8"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03500-8" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Single-cell transcriptomics can provide quantitative molecular
              signatures for large, unbiased samples of the diverse cell types
              in the brain1-3. With the proliferation of multi-omics datasets,
              a major challenge is to validate and integrate results into a
              biological understanding of cell-type organization. Here we
              generated transcriptomes and epigenomes from more than 500,000
              individual cells in the mouse primary motor cortex, a structure
              that has an evolutionarily conserved role in locomotion. We
              developed computational and statistical methods to integrate
              multimodal data and quantitatively validate cell-type
              reproducibility. The resulting reference atlas-containing over 56
              neuronal cell types that are highly replicable across analysis
              methods, sequencing technologies and modalities-is a
              comprehensive molecular and genomic account of the diverse
              neuronal and non-neuronal cell types in the mouse primary motor
              cortex. The atlas includes a population of excitatory neurons
              that resemble pyramidal cells in layer 4 in other cortical
              regions4. We further discovered thousands of concordant marker
              genes and gene regulatory elements for these cell types. Our
              results highlight the complex molecular regulation of cell types
              in the brain and will directly enable the design of reagents to
              target specific cell types in the mouse primary motor cortex for
              functional analysis.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Cover%207%20October%202021-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Cover%207%20October%202021.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Cover 7 October 2021.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Li2021-er" class="col-sm-8">
        <!-- Title -->
        <div class="title">An atlas of gene regulatory elements in adult mouse cerebrum</div>
        <!-- Author -->
        <div class="author">
            

            Yang Eric Li, Sebastian Preissl, Xiaomeng Hou, Ziyang Zhang, Kai Zhang, Yunjiang Qiu, Olivier B Poirion, Bin Li, Joshua Chiou, <em><strong>Hanqing Liu</strong></em>, Antonio Pinto-Duarte, Naoki Kubo, Xiaoyu Yang, Rongxin Fang, Xinxin Wang, Jee Yun Han, Jacinta Lucero, Yiming Yan, Michael Miller, Samantha Kuan, David Gorkin, Kyle J Gaulton, Yin Shen, Michael Nunn, Eran A Mukamel, M Margarita Behrens, Joseph R Ecker, and Bing Ren</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nature</em>, Oct 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.nature.com/articles/s41586-021-03604-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41586-021-03604-1"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-021-03604-1" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>The mammalian cerebrum performs high-level sensory perception,
              motor control and cognitive functions through highly specialized
              cortical and subcortical structures1. Recent surveys of mouse and
              human brains with single-cell transcriptomics2-6 and
              high-throughput imaging technologies7,8 have uncovered hundreds
              of neural cell types distributed in different brain regions, but
              the transcriptional regulatory programs that are responsible for
              the unique identity and function of each cell type remain
              unknown. Here we probe the accessible chromatin in more than
              800,000 individual nuclei from 45 regions that span the adult
              mouse isocortex, olfactory bulb, hippocampus and cerebral nuclei,
              and use the resulting data to map the state of 491,818 candidate
              cis-regulatory DNA elements in 160 distinct cell types. We find
              high specificity of spatial distribution for not only excitatory
              neurons, but also most classes of inhibitory neurons and a subset
              of glial cell types. We characterize the gene regulatory
              sequences associated with the regional specificity within these
              cell types. We further link a considerable fraction of the
              cis-regulatory elements to putative target genes expressed in
              diverse cerebral cell types and predict transcriptional
              regulators that are involved in a broad spectrum of molecular and
              cellular pathways in different neuronal and glial cell
              populations. Our results provide a foundation for comprehensive
              analysis of gene regulatory programs of the mammalian brain and
              assist in the interpretation of noncoding risk variants
              associated with various neurological diseases and traits in
              humans.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Elife-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Elife-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Elife-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Elife.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Elife.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Farhy-Tselnicker2021-lt" class="col-sm-8">
        <!-- Title -->
        <div class="title">Activity-dependent modulation of synapse-regulating genes in
              astrocytes</div>
        <!-- Author -->
        <div class="author">
            

            Isabella Farhy-Tselnicker, Matthew M Boisvert, <em><strong>Hanqing Liu</strong></em>, Cari Dowling, Galina A Erikson, Elena Blanco-Suarez, Chen Farhy, Maxim N Shokhirev, Joseph R Ecker, and Nicola J Allen</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Elife</em>, Sep 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://elifesciences.org/articles/70514" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.7554/eLife.70514"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.7554/eLife.70514" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Astrocytes regulate the formation and function of neuronal
              synapses via multiple signals; however, what controls regional
              and temporal expression of these signals during development is
              unknown. We determined the expression profile of astrocyte
              synapse-regulating genes in the developing mouse visual cortex,
              identifying astrocyte signals that show differential temporal and
              layer-enriched expression. These patterns are not intrinsic to
              astrocytes, but regulated by visually evoked neuronal activity,
              as they are absent in mice lacking glutamate release from
              thalamocortical terminals. Consequently, synapses remain
              immature. Expression of synapse-regulating genes and synaptic
              development is also altered when astrocyte signaling is blunted
              by diminishing calcium release from astrocyte stores.
              Single-nucleus RNA sequencing identified groups of astrocytic
              genes regulated by neuronal and astrocyte activity, and a
              cassette of genes that show layer-specific enrichment. Thus, the
              development of cortical circuits requires coordinated signaling
              between astrocytes and neurons, highlighting astrocytes as a
              target to manipulate in neurodevelopmental disorders.</p>
        </div>
    </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Nature%20Communications-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Nature%20Communications-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Nature%20Communications-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Nature%20Communications.png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Nature Communications.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Ruf-Zamojski2021-cs" class="col-sm-8">
        <!-- Title -->
        <div class="title">Single nucleus multi-omics regulatory landscape of the murine
              pituitary</div>
        <!-- Author -->
        <div class="author">
            

            Frederique Ruf-Zamojski, Zidong Zhang, Michel Zamojski, Gregory R Smith, Natalia Mendelev, <em><strong>Hanqing Liu</strong></em>, German Nudelman, Mika Moriwaki, Hanna Pincas, Rosa Gomez Castanon, Venugopalan D Nair, Nitish Seenarine, Mary Anne S Amper, Xiang Zhou, Luisina Ongaro, Chirine Toufaily, Gauthier Schang, Joseph R Nery, Anna Bartlett, Andrew Aldridge, Nimisha Jain, Gwen V Childs, Olga G Troyanskaya, Joseph R Ecker, Judith L Turgeon, Corrine K Welt, Daniel J Bernard, and Stuart C Sealfon</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Nat. Commun.</em>, May 2021
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1038/s41467-021-22859-w"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-021-22859-w" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>To provide a multi-omics resource and investigate transcriptional
              regulatory mechanisms, we profile the transcriptome, chromatin
              accessibility, and methylation status of over 70,000 single
              nuclei (sn) from adult mouse pituitaries. Paired snRNAseq and
              snATACseq datasets from individual animals highlight a continuum
              between developmental epigenetically-encoded cell types and
              transcriptionally-determined transient cell states.
              Co-accessibility analysis-based identification of a putative Fshb
              cis-regulatory domain that overlaps the fertility-linked
              rs11031006 human polymorphism, followed by experimental
              validation illustrate the use of this resource for hypothesis
              generation. We also identify transcriptional and chromatin
              accessibility programs distinguishing each major cell type.
              Regulons, which are co-regulated gene sets sharing binding sites
              for a common transcription factor driver, recapitulate cell type
              clustering. We identify both cell type-specific and sex-specific
              regulons that are highly correlated with promoter accessibility,
              but not with methylation state, supporting the centrality of
              chromatin accessibility in shaping cell-defining transcriptional
              programs. The sn multi-omics atlas is accessible at
              snpituitaryatlas.princeton.edu.</p>
        </div>
    </div>
</div>
</li>
</ol>

<h2 class="year">2020</h2>
<ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/JCI%20Insight%205-23-cover-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/JCI%20Insight%205-23-cover-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/JCI%20Insight%205-23-cover-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/JCI%20Insight%205-23-cover.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="JCI Insight 5-23-cover.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Cui2020-vp" class="col-sm-8">
        <!-- Title -->
        <div class="title">Mutations of MAP1B encoding a microtubule-associated
              phosphoprotein cause sensorineural hearing loss</div>
        <!-- Author -->
        <div class="author">
            

            Limei Cui, Jing Zheng, Qiong Zhao, Jia-Rong Chen, <em><strong>Hanqing Liu</strong></em>, Guanghua Peng, Yue Wu, Chao Chen, Qiufen He, Haosong Shi, Shankai Yin, Rick A Friedman, Ye Chen, and Min-Xin Guan</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>JCI Insight</em>, Dec 2020
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://insight.jci.org/articles/view/136046" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1172/jci.insight.136046"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1172/jci.insight.136046" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>The pathophysiology underlying spiral ganglion cell
              defect-induced deafness remains elusive. Using the whole exome
              sequencing approach, in combination with functional assays and a
              mouse disease model, we identified the potentially novel
              deafness-causative MAP1B gene encoding a highly conserved
              microtubule-associated protein. Three novel heterozygous MAP1B
              mutations (c.4198A&gt;G, p.1400S&gt;G; c.2768T&gt;C, p.923I&gt;T; c.5512T&gt;C,
              p.1838F&gt;L) were cosegregated with autosomal dominant inheritance
              of nonsyndromic sensorineural hearing loss in 3 unrelated Chinese
              families. Here, we show that MAP1B is highly expressed in the
              spiral ganglion neurons in the mouse cochlea. Using otic sensory
              neuron-like cells, generated by pluripotent stem cells from
              patients carrying the MAP1B mutation and control subject, we
              demonstrated that the p.1400S&gt;G mutation caused the reduced
              levels and deficient phosphorylation of MAP1B, which are involved
              in the microtubule stability and dynamics. Strikingly, otic
              sensory neuron-like cells exhibited disturbed dynamics of
              microtubules, axonal elongation, and defects in
              electrophysiological properties. Dysfunctions of these derived
              otic sensory neuron-like cells were rescued by genetically
              correcting MAP1B mutation using CRISPR/Cas9 technology.
              Involvement of MAP1B in hearing was confirmed by audiometric
              evaluation of Map1b heterozygous KO mice. These mutant mice
              displayed late-onset progressive sensorineural hearing loss that
              was more pronounced in the high frequencies. The spiral ganglion
              neurons isolated from Map1b mutant mice exhibited the deficient
              phosphorylation and disturbed dynamics of microtubules. Map1b
              deficiency yielded defects in the morphology and
              electrophysiology of spiral ganglion neurons, but it did not
              affect the morphologies of cochlea in mice. Therefore, our data
              demonstrate that dysfunctions of spiral ganglion neurons induced
              by MAP1B deficiency caused hearing loss.</p>
        </div>
    </div>
</div>
</li></ol>

<h2 class="year">2019</h2>
<ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 abbr"></div>

    <!-- Entry bib key -->
    <div id="Zheng2019-eh" class="col-sm-8">
        <!-- Title -->
        <div class="title">New SNP variants of MARVELD2 (DFNB49) associated with
              non-syndromic hearing loss in Chinese population</div>
        <!-- Author -->
        <div class="author">
            

            Jing Zheng, Wen-Fang Meng, Chao-Fan Zhang, <em><strong>Hanqing Liu</strong></em>, Juan Yao, Hui Wang, Ye Chen, and Min-Xin Guan</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>J. Zhejiang Univ. Sci. B</em>, Feb 2019
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6380993/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1631/jzus.B1700185"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1631/jzus.B1700185" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>Non-syndromic hearing loss (NSHL) is a common defect in humans.
              Variants of MARVELD2 at the DFNB49 locus have been shown to cause
              bilateral, moderate to profound NSHL. However, the role of
              MARVELD2 in NSHL susceptibility in the Chinese population has not
              been studied. Here we conducted a case-control study in an
              eastern Chinese population to profile the spectrum and frequency
              of MARVELD2 variants, as well as the association of MARVELD2 gene
              variants with NSHL. Our results showed that variants identified
              in the Chinese population are significantly different from those
              reported in Slovak, Hungarian, and Czech Roma, as well as
              Pakistani families. We identified 11 variants in a cohort of 283
              NSHL cases. Through Sanger sequencing and bioinformatics
              analysis, we found that c.730G&gt;A variant has detrimental effects
              in the eastern Chinese population, and may have relatively high
              correlation with NSHL pathogenicity.</p>
        </div>
    </div>
</div>
</li></ol>

<h2 class="year">2016</h2>
<ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
    <div class="col-sm-2 preview">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/Molecular%20and%20Cellular%20Biology,%20Volume%2036,%20Issue%2014%20(2016)-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/Molecular%20and%20Cellular%20Biology,%20Volume%2036,%20Issue%2014%20(2016)-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/Molecular%20and%20Cellular%20Biology,%20Volume%2036,%20Issue%2014%20(2016)-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/Molecular%20and%20Cellular%20Biology,%20Volume%2036,%20Issue%2014%20(2016).png" class="preview z-depth-1 rounded" width="auto" height="auto" alt="Molecular and Cellular Biology, Volume 36, Issue 14 (2016).png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>

    <!-- Entry bib key -->
    <div id="Jiang2016-tr" class="col-sm-8">
        <!-- Title -->
        <div class="title">A Hypertension-Associated tRNAAla Mutation Alters tRNA
              Metabolism and Mitochondrial Function</div>
        <!-- Author -->
        <div class="author">
            

            Pingping Jiang, Meng Wang, Ling Xue, Yun Xiao, Jialing Yu, Hui Wang, Juan Yao, Hao Liu, Yanyan Peng, <em><strong>Hanqing Liu</strong></em>, Haiying Li, Ye Chen, and Min-Xin Guan</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
            <em>Mol. Cell. Biol.</em>, Jul 2016
        </div>
        <div class="periodical">
            
        </div>

        <!-- Links/Buttons -->
        <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
            <a href="https://www.tandfonline.com/doi/full/10.1128/MCB.00199-16" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
        </div>
        
        <div class="badges">
            <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1128/MCB.00199-16"></span>
            <span class="__dimensions_badge_embed__" data-doi="10.1128/MCB.00199-16" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span>
        </div>

        <!-- Hidden abstract block -->
        <div class="abstract hidden">
            <p>In this report, we investigated the pathophysiology of a novel
              hypertension-associated mitochondrial tRNA(Ala) 5655A
              →G (m.5655A →G) mutation. The
              destabilization of a highly conserved base pairing (A1-U72) at
              the aminoacyl acceptor stem by an m.5655A →G
              mutation altered the tRNA(Ala) function. An in vitro processing
              analysis showed that the m.5655A →G mutation reduced
              the efficiency of tRNA(Ala) precursor 5’ end cleavage catalyzed
              by RNase P. By using cybrids constructed by transferring
              mitochondria from lymphoblastoid cell lines derived from a
              Chinese family into mitochondrial DNA (mtDNA)-less (ρ(o))
              cells, we showed a 41% reduction in the steady-state level of
              tRNA(Ala) in mutant cybrids. The mutation caused an improperly
              aminoacylated tRNA(Ala), as suggested by aberrantly aminoacylated
              tRNA(Ala) and slower electrophoretic mobility of mutated tRNA. A
              failure in tRNA(Ala) metabolism contributed to variable
              reductions in six mtDNA-encoded polypeptides in mutant cells,
              ranging from 21% to 37.5%, with an average of a 29.1%
              reduction, compared to levels of the controls. The impaired
              translation caused reduced activities of mitochondrial
              respiration chains. Furthermore, marked decreases in the levels
              of mitochondrial ATP and membrane potential were observed in
              mutant cells. These caused increases in the production of
              reactive oxygen species in the mutant cybrids. The data provide
              evidence for the association of the tRNA(Ala) 5655A →G mutation with hypertension.</p>
        </div>
    </div>
</div>
</li></ol>


</div>

          </article>

        </div>

    </div>

    <!-- Footer -->    <footer class="sticky-bottom mt-5">
      <div class="container">
        © Copyright 2023 Hanqing  Liu. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-RC1D7R1NBW"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){ window.dataLayer.push(arguments); }
    gtag('js', new Date());
    gtag('config', 'G-RC1D7R1NBW');
  </script>
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
